src/components/Ideas/CompIdeaDetail.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, useEffect, useState } from "react"
+import { IdeaContext } from "./IdeaProvider"
+import { useParams, useHistory } from "react-router-dom"
+import "./Idea.css"
+
+export const CompIdeaDetail = (props) => {
```

Comment:
If you're passing a component props, we want to destructure the keys or use propTypes just so that anyone using the component knows what props this component takes

```suggestion
export const CompIdeaDetail = ({
idea, 
}) => {
```

Reply:
Might have to change the variable name since you have something called idea.